docs/pages/docs/apis/access-control.mdx

```diff
@@ -20,244 +20,171 @@ export default config({
 });
 ```
 
-This document covers the complete access control API.
-For a guide on how to use this API to apply common patterns please see the [access control guide](../guides/access-control).
+!> This document covers the complete access control API.
+Our [access control guide](../guides/access-control) shows you how to apply this API to common patterns.
 
 ## List Access Control
 
-Keystone allows you to set up access control on a per-list basis.
-The default access control is to allow all operations for all users.
-Access control is applied to the generated CRUD (**c**reate, **r**ead, **u**pdate, **d**elete) queries and mutations in the [GraphQL API](./graphql).
-Access control is applied before any [hooks](./hooks) are executed.
+Keystone lets you apply access control on a per-list basis.
+By default it allows all operations for all users.
+Access control is applied to the generated CRUD (**c**reate, **r**ead, **u**pdate, **d**elete) operations and mutations in the [GraphQL API](./graphql), and is applied before any [hooks](./hooks) are executed.
 
-You can specify access control using either **concise** or **verbose** syntax.
-Verbose syntax provides a separate access control rule for each operation type; `create`, `read`, `update`, and `delete`.
-Concise syntax provides a single access control rule which is used for all operation types.
+List-level access control can be specified in three ways, of increasing power.
```

Comment:
Probs a nitpick, but 'increasing power' doesn't explain what this means to me.

Reply:
Sentence removed 👍